### PR TITLE
Connected node server to the hosted MongoLab DB.

### PR DESCRIPTION
**Pull request by jhennin**
You must have the 'credentials.js' file for this to work.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -8,19 +8,32 @@ var express = require('express');
 var mongoose = require('mongoose');
 var passport = require('passport');
 var config = require('./config/config');
+var credentials = require('./credentials.js');
```
Comment (bergera) [inline server.js]: This needs to be checked to ensure the file exists. Look into [`fs.stat()`](https://nodejs.org/api/fs.html#fs_file_system).

```
$ npm start

> ceg-4981-team-projects@1.0.0 start /home/andrew/projects/senior-design
> nodemon server.js

[nodemon] 1.8.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node server.js`
module.js:328
    throw err;
    ^

Error: Cannot find module './credentials.js'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/andrew/projects/senior-design/server.js:11:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by bergera** [comment]
A better place to load this would be `./config/env/production.js`.

**Comment by jhennin** [comment]
I avoided doing anything with the env files because those are exposed on our repo.  But I can change that if its not the case.

**Comment by bergera** [comment]
Let's come back to this when we know we want to run in production mode.

**Pull request closed without merging**